src/App.test.js

```diff
@@ -2,6 +2,9 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 
+// Mocked because of UnhandledPromiseRejectionWarning in Events
+jest.mock('./Events/Events', () => () => <div />);
```

Comment:
If we add a `catch` to that promise, could this then be removed?

Reply:
Agreed, will fix in feature/events-page